e2e/commands/start/start_test.go

```diff
@@ -463,4 +463,60 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 			Expect(session.Out.Contents()).To(ContainSubstring("YOUR ANALYSIS HAD FINISHED WITHOUT ANY VULNERABILITY!"))
 		})
 	})
+
+	When("--custom-path-rules is passed", func() {
+		customRulesJson := testutil.GinkgoCreateTmpFile("*.json")
+		writeJsonFile(customRulesJson)
+
+		BeforeEach(func() {
+
+			flags = map[string]string{
+				testutil.StartFlagProjectPath:     testutil.JavaExample1,
+				testutil.StartFlagCustomRulesPath: customRulesJson,
+			}
+		})
+
+		It("Checks if the custom rules path property was set", func() {
+			Expect(session.Out.Contents()).To(ContainSubstring(`\"custom_rules_path\": \"%s\"`, testutil.NormalizePathToAssertInJSON(customRulesJson)))
+			Eventually(session.Wait(5 * time.Minute.Minutes()).Out).Should(gbytes.Say(`Language: Java`))
+			Eventually(session.Wait(5 * time.Minute.Minutes()).Out).Should(gbytes.Say(`Severity: LOW`))
+			Eventually(session.Wait(5 * time.Minute.Minutes()).Out).Should(gbytes.Say(`Confidence: LOW`))
+			Eventually(session.Wait(5 * time.Minute.Minutes()).Out).Should(gbytes.Say(`RuleID: HS-JAVA-99999999999`))
+			Eventually(session.Wait(5 * time.Minute.Minutes()).Out).Should(gbytes.Say(`Details: Teste QA`))
+			Eventually(session.Wait(5 * time.Minute.Minutes()).Out).Should(gbytes.Say(`Type: Vulnerability`))
```

Comment:
Create a variable with the timout and use the reference:
```go
timeout := 5 * time.Minute
Eventually(session.Wait(timeout).Out...
```
And btw you don't need to call `Minute.Minutes()` method, just use `N * time.Minute`.